contracts/truefi2/LoanFactory2.sol

```diff
@@ -154,4 +152,10 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
         borrowingMutex = _mutex;
         emit BorrowingMutexChanged(_mutex);
     }
+
+    function setLoanTokenImplementation(ILoanToken2 _implementation) external onlyAdmin {
+        require(address(_implementation) != address(0), "LoanFactory: Cannot set loan token implementation to address(0)");
```

Comment:
Following https://github.com/trusttoken/smart-contracts/pull/930#discussion_r701707523, let's use `zero address` instead of `address(0)` in error messages and test naming.

Reply:
How about we create separate PR where we change all instances of `address(0)` (we have quite a few in tests) to `zero address`?

Reply:
Sure, that sounds fine to me!

The naming isn't a big deal, so I'd put this at the bottom of the backlog :)

Reply:
Added the task to the backlog.